packages/front-end/pages/report/[rid].tsx

```diff
@@ -376,6 +376,7 @@ export default function ReportPage() {
                 variations={variations}
                 key={report.args.dimension}
                 statsEngine={report.args.statsEngine}
+                pValueCorrection={settings.pValueCorrection ?? 'none'}
```

Comment:
we might want to use a nullish default value instead of the string "none". Perhaps "".

Reply:
Why is that better than being explicit? Not sure I understand.

Reply:
There's a fair amount of JIT coercion with the magic value of "none", such as:
`((pValueCorrection ?? 'none') !== 'none') `
It's fine I suppose, but generally when I think of an enum that allows for an "off" state, I'd like to be able to do a simpler check such as:
`if(pValueCorrection)`

You could even use a nullable type such as `"option1" | "option2" | null`, and that way we don't need to store "none" in the database.

Reply:
That makes sense! Thanks for explaining it to me. I slightly prefer "" because I think having a value in the database is better than relying on null values. Then null is informative in the data for debugging/etc.